components/menu/index.css

```diff
@@ -782,3 +754,32 @@ governing permissions and limitations under the License.
     }    
   }
 }
+
+/* Disabled menu items */
+.spectrum-Menu-item.is-disabled,
+.spectrum-Menu-item[aria-disabled="true"] {
+  background-color: transparent;
+
+  .spectrum-Menu-itemLabel,
+  .spectrum-Menu-sectionHeading {
+    color: var(--highcontrast-menu-item-color-disabled, var(--mod-menu-item-label-content-color-disabled, var(--spectrum-menu-item-label-content-color-disabled)));
+  }
+  
+  .spectrum-Menu-itemDescription {
+    color: var(--highcontrast-menu-item-color-disabled, var(--mod-menu-item-description-color-disabled, var(--spectrum-menu-item-description-color-disabled)));
+  }
+  
+  .spectrum-Menu-itemIcon {
+    fill: var(--highcontrast-menu-item-color-disabled, var(--mod-menu-item-label-icon-color-disabled, var(--spectrum-menu-item-label-icon-color-disabled)));
+    color: var(--highcontrast-menu-item-color-disabled, var(--mod-menu-item-label-icon-color-disabled, var(--spectrum-menu-item-label-icon-color-disabled)));
+  }
+
+  &:hover {
+    cursor: default;
+
+    .spectrum-Menu-itemIcon {
```

Comment:
Hi, sorry to revive this issue, but this implementation doesn't work with our API in SWC. The chevron is implemented differently than the other icons that can be slotted into the menu item, so I think the class `.spectrum-Menu-chevron` would need specification for being disabled, as well, in order to receive styling. Please look [here](https://studio.webcomponents.dev/edit/collection/fO75441E1Q5ZlI0e9pgq/FikFeTXNsYhxAVmCz2f4/src/index.ts?branch=submenu-disabled%40yOBt1947wIZCL9LXYVLsyGBLOok1&p=stories) to see the difference between an icon and the chevron. The icon has disabled styling, but the chevron still doesn't.

Reply:
I took a look at this and tested out added CSS with this draft PR https://github.com/adobe/spectrum-css/pull/2550/files#diff-3b8d6209262a97e225fba580d5a675267ced29e23aadabdcc2cb73332bf11d94 , but in looking at it further I'm not sure if this extra selector is something we should add on our end? Your thoughts @pfulton ? SWC seems to have diverged from how we use our CSS. On our end, the `.spectrum-Menu-itemIcon` class is a generic one applied to ALL menu item icons, including the workflow icon, the collapsible chevron, the drill-in chevron, and the checkmark. Can SWC also apply this class or these styles to all icons?

Reply:
> Separately, I'm never sure if I'm working from latest in the CSS public documentations, but can you apply `is-disabled` to one of the Menu Items in [this story](https://opensource.adobe.com/spectrum-css/preview/?path=/story/components-menu--with-value-and-actions)?  We'd love to see CSS applying the disabled text color to that "value" content as well, here.

In PR #2579 I've updated the Storybook templates with that increased coverage and have added CSS for the "value" text to use the disabled content color.